ios/RNSScreenStack.m

```diff
@@ -285,6 +285,12 @@ - (void)setModalViewControllers:(NSArray<UIViewController *> *)controllers
       for (NSUInteger i = changeRootIndex; i < controllers.count; i++) {
         UIViewController *next = controllers[i];
         BOOL lastModal = (i == controllers.count - 1);
+
+        if (@available(iOS 13.0, *)) {
```

Comment:
This also does not matter I think as soon appstore will stop accepting builds made with prev versions of xcode (if that isn't the case already) but for consistency can we add `#ifdef __IPHONE_13_0` to wrap this call much like we do in other cases where we do version checks